### PR TITLE
add cornell-eas-data-lake

### PR DESCRIPTION
**Pull request by marty-sullivan**
Adding Cornell Earth & Atmospheric Sciences Data Lake

### CODE REVIEW

File: datasets/cornell-eas-data-lake.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+Name: 'Cornell EAS Data Lake'
+Description: >
+  Earth & Atmospheric Sciences at Cornell University has created a public data
+  lake of climate data. The data is stored in columnar storage formats (ORC) to
+  make it straightforward to query using standard tools like Amazon
+  Athena or Apache Spark. The data itself is originally intended to be used for
+  building decision support tools for farmers and digital agriculture. The first
+  dataset is the historical NDFD / NDGD data distributed by NCEP / NOAA / NWS.
```
Comment (jflasher) [inline datasets/cornell-eas-data-lake.yml]: Could you please define NDFD and NDGD?

File: datasets/cornell-eas-data-lake.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Documentation: https://datalake.eas.cornell.edu/
+Contact: digitalag@cornell.edu
+UpdateFrequency: Hourly
+Tags:
```
Comment (jflasher) [inline datasets/cornell-eas-data-lake.yml]: Please add 'aws-pds' as a tag as well.

File: datasets/cornell-eas-data-lake.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  - weather
+
+License: https://datalake.eas.cornell.edu/license.txt
+Resources:
+  - Description: Cornell EAS Data Lake
+    ARN: arn:aws:s3:::cornell-eas-data-lake
+    Region: us-east-2
+    Type: S3 Bucket
+
+  - Description: Cornell EAS Data Lake Notifications
```
Comment (jflasher) [inline datasets/cornell-eas-data-lake.yml]: Could you please add a little more description of the difference between this and the other SNS topic?

File: datasets/cornell-eas-data-lake.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    ARN: arn:aws:sns:us-east-2:003709786761:cornell-eas-data-lake-human
+    Region: us-east-2
+    Type: SNS Topic
+    
+  - Description: Cornell EAS Data Lake Automation Notifications
+    ARN: arn:aws:sns:us-east-2:003709786761:cornell-eas-data-lake
+    Region: us-east-2
+    Type: SNS Topic
+    
+#DataAtWork:
```
Comment (jflasher) [inline datasets/cornell-eas-data-lake.yml]: If you want to add https://datalake.eas.cornell.edu/ as an example usage here, I think that'd be great!

**Comment by marty-sullivan** [comment]
Thanks for looking things over, Joe. Feel free to let me know if you'd like anything else added!

**Comment by marty-sullivan** [comment]
The [website](https://datalake.eas.cornell.edu) is all set. Feel free to merge 👍

**Pull request merged**